test/language/statements/class/decorator/syntax/valid/decorator-call-expr-identifier-reference-yield.js

```diff
@@ -1,15 +1,19 @@
 // This file was procedurally generated from the following sources:
 // - src/decorator/decorator-call-expr-identifier-reference-yield.case
-// - src/decorator/syntax/valid/cls-decl-decorators-valid-syntax.template
+// - src/decorator/syntax/valid/cls-element-decorators-valid-syntax.template
```

Comment:
This doesn't seem right. I remember somewhat recently there was a case where two test generators raced to produce the same file, which I believe was fixed. Is it possible that this branch is old enough that it doesn't have that fix, or maybe there's another similar case that wasn't caught yet?

Reply:
Yep, that was the case! A rebase, clean, and re-build fixed it. Thanks for catching it.